ethereum-studio-docker-standalone-ubuntu/azuredeploy.parameters.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "value": "userName"
+    },
+    "adminPassword": {
+      "value": "password"
```

Comment:
Placeholder parameters are needed here to pass the CI. See https://github.com/Azure/azure-quickstart-templates/tree/master/1-CONTRIBUTION-GUIDE#travis-ci